src/core/Suite.js

```diff
@@ -14,6 +14,8 @@ getJasmineRequireObj().Suite = function(j$) {
     this.beforeAllFns = [];
     this.afterAllFns = [];
 
+    this.timer = attrs.timer || new j$.Timer();
```

Comment:
This should fallback to a dummy timer and force injection in a real suite. It might be time to pull [noopTimer](https://github.com/jasmine/jasmine/blob/master/src/html/HtmlReporter.js#L3) out of HtmlReporter so it can be shared.